frontend/src/util/db.ts

```diff
@@ -100,23 +105,28 @@ export class IndexedDBCache {
 		GetSession: moment.duration(15, 'minutes').asMilliseconds(),
 	}
 	getItem = async function (key: { operation: string; variables: any }) {
-		const result = await db.apollo
-			.where('key')
-			.equals(JSON.stringify(key))
-			.first()
-		if (result) {
-			if (IndexedDBCache.expiryMS[key.operation]) {
-				if (
-					moment().diff(moment(result.created)) >=
-					IndexedDBCache.expiryMS[key.operation]
-				) {
-					db.apollo.delete(result.key)
-					return null
+		try {
+			const result = await db.apollo
+				.where('key')
+				.equals(JSON.stringify(key))
+				.first()
+			if (result) {
+				if (IndexedDBCache.expiryMS[key.operation]) {
+					if (
+						moment().diff(moment(result.created)) >=
+						IndexedDBCache.expiryMS[key.operation]
+					) {
+						db.apollo.delete(result.key)
+						return null
+					}
 				}
+				db.apollo.update(result.key, { updated: moment().format() })
 			}
-			db.apollo.update(result.key, { updated: moment().format() })
+			return result?.data ?? null
+		} catch {
```

Comment:
Have had trouble reproducing - I wanted to limit this to `UnknownError` as in https://app.highlight.io/1/errors/eN8JNCVob09aKNV1cSrtrqwcn0JJ, but can't confirm it works